cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -335,20 +330,27 @@ func (h eventsRPCHandler) getEvents(request GetEventsRequest) (GetEventsResponse
 	}, nil
 }
 
+func eventTypeFromXDR(xdrType xdr.ContractEventType) string {
+	switch xdrType {
+	case xdr.ContractEventTypeSystem:
+		return EventTypeSystem
+	case xdr.ContractEventTypeContract:
+		return EventTypeContract
+	case xdr.ContractEventTypeDiagnostic:
+		return EventTypeDiagnostic
+	}
+	return ""
+}
```

Comment:
can/should this just be a static map rather than a runtime lookup?

Reply:
That’s a good idea, I will do that